packages/bpk-component-barchart/src/BpkBarchart.js

```diff
@@ -45,17 +47,88 @@ const getClassName = cssModules(STYLES);
 const spacing = remToPx(spacingXs);
 const lineHeight = remToPx(lineHeightSm);
 
-const getMaxYValue = (dataPoints, yScaleDataKey, outlierPercentage) => {
+const getMaxYValue = (dataPoints: Function, outlierPercentage: ?number) => {
```

Comment:
Two questions: 

1. How come `yScaleDataKey` was removed as an argument?
1. Is `Function` the correct type for `dataPoints`? Below this we call `.reduce` and `.length` on it, which indicates it's an array of numbers.

Reply:
1. It was not being used in the function so was removed

2. I tried to change the type but it complained due to the underlying third party lib being used saying that they weren't functions that exist 🤔

Reply:
Ah, weird.. so `Array<number>` didn't work? 

If not, what if you defined it manually like this:

```
dataPoints: { reduce: () => number, length: () => number }
```